### YamlMime:UniversalReference
api_name: []
items:
- children:
  - azure.mgmt.resource.policy.v2018_03_01.models.error_response_py3.ErrorResponse
  - azure.mgmt.resource.policy.v2018_03_01.models.error_response_py3.ErrorResponseException
  fullName: azure.mgmt.resource.policy.v2018_03_01.models.error_response_py3
  langs:
  - python
  module: azure.mgmt.resource.policy.v2018_03_01.models.error_response_py3
  name: error_response_py3
  type: module
  uid: azure.mgmt.resource.policy.v2018_03_01.models.error_response_py3
references:
- fullName: azure.mgmt.resource.policy.v2018_03_01.models.error_response_py3.ErrorResponse
  isExternal: false
  name: ErrorResponse
  parent: azure.mgmt.resource.policy.v2018_03_01.models.error_response_py3
  uid: azure.mgmt.resource.policy.v2018_03_01.models.error_response_py3.ErrorResponse
- fullName: azure.mgmt.resource.policy.v2018_03_01.models.error_response_py3.ErrorResponseException
  isExternal: false
  name: ErrorResponseException
  parent: azure.mgmt.resource.policy.v2018_03_01.models.error_response_py3
  uid: azure.mgmt.resource.policy.v2018_03_01.models.error_response_py3.ErrorResponseException
